addons/dynamishelper/dynamishelper.lua

```diff
@@ -24,408 +24,427 @@
+}
+
+function printHelp(command,val)
+    if not command and not val then
+        windower.add_to_chat(159,'\nDynamisHelper v2.0')
+        windower.add_to_chat(159,'dh size [number]: Change the font size.')
+        windower.add_to_chat(159,'dh font [Arial, Tahoma, Times "Open Sans" ...]: Change the font.')
+        windower.add_to_chat(159,'dh posx [pixel]: Position on the X axis in pixel.')
+        windower.add_to_chat(159,'dh posy [pixel]: Position on the Y axis in pixel.')
+        windower.add_to_chat(159,'dh bgcolor [Red] [Green] [Blue] - Background color - Example: 255 0 0 is red')
+        windower.add_to_chat(159,'dh opacity bg_alpha: Opacity (0-255) of the background.')
+        windower.add_to_chat(159,'dh visible: Toggle addon window.')
+        windower.add_to_chat(159,'dh bold: Toggle bold text.')
+        windower.add_to_chat(159,'dh padding [size]: Padding of the text window.')
+        windower.add_to_chat(159,'dh stroke [width]: Stroke width of the text.')
+        windower.add_to_chat(159,'dh stopacity [0-100]%: Stroke opacity.')
+        windower.add_to_chat(159,'dh stcolor [Red] [Green] [Blue] - Stroke color - Example: 255 0 0 is red')
+
+        -- compatibility commands ---------------------------------------------
+        windower.add_to_chat(159,'dh ll create: Creates a light luggage profile to lot all dynamis currency.')
+        -----------------------------------------------------------------------
+        windower.add_to_chat(159,'dh save: save your current settings.')
+    else
+        local m = string.format(help[command],val)
+        windower.add_to_chat(159,'\nDynamisHelper v2.0')
+        windower.add_to_chat(159,m)
+    end
 end
 
-windower.register_event('unload',function ()
- 	windower.text.delete('dynamis_box')
- 	windower.text.delete('proc_box')
+-------------------------------------------------------------------------------
+-- Process options and save settings ------------------------------------------
+-------------------------------------------------------------------------------
+windower.register_event('addon command',function (command, ...)
+    command = command and command:lower() or 'help'
+    local options = {...}
+    if command == "help" then
+        printHelp()
+        return
+    elseif command:lower() == "visible" then
+        if window:visible() then
+            window:hide()
+        else
+            window:show()
+        end
+    elseif command == "size" then
+        if options[1] and tonumber(options[1]) then
+            window:size(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,settings.window.text.size)
+        end
+    elseif command == "font" then
+        if options[1] then
+            window:font(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,settings.window.text.font)
+        end
+    elseif command == "posx" then
+        if options[1] then
```

Comment:
I think this needs `tonumber` as well? Same for some of the next ones. Maybe just go over these once more to make sure.